Library/Homebrew/dependable.rb

```diff
@@ -1,7 +1,7 @@
 require "options"
 
 module Dependable
-  RESERVED_TAGS = [:build, :optional, :recommended, :run, :linked]
+  RESERVED_TAGS = [:build, :optional, :recommended, :run]
```

Comment:
I mentioned it elsewhere, but for completeness sake: This can only be reverted once all `depends_on "giraffe" => :linked` lines have been eliminated from formulae, i.e. after Homebrew/homebrew-core#3370 is merged. (And I haven't checked, but hopefully those lines haven't also trickled into other official taps or even taps not maintained by us.)